client/wfnews-war/src/main/angular/src/app/components/dashboard-component/widgets/highlights-widget/highlights-widget.component.ts

```diff
@@ -139,4 +141,22 @@ export class HighlightsWidgetComponent implements OnInit {
       fireCentres: fireCentreTags.map(tag => tag.name)  // Array of fire centre names
     };
   }
+
+  // Use CapacitorHttp if on mobile to avoid CORS issues
+  private processRequest<T>(url: string): Observable<T> {   
```

Comment:
I assume we're always using capacitorhttp elsewhere already?  

Is  method something that should live in a more gereral context?  Will other components need to use this same technique?

Reply:
The more I think about this, the more I'd like to see logic like this in a service class.

Reply:
Thanks, I moved the getRequest function to the common utility service now, and added some formatting/linting changes in the files I was working on.